src/Raven.Client/Documents/Session/SessionTimeSeriesBase.cs

```diff
@@ -122,6 +122,11 @@ public void Delete(DateTime? from = null, DateTime? to = null)
             {
                 Session.Defer(new TimeSeriesBatchCommandData(DocId, Name, appends: null, deletes: new List<TimeSeriesOperation.DeleteOperation> { op }));
             }
+
+            if (Session.TimeSeriesByDocId.TryGetValue(DocId, out var cache))
+            {
+                cache.Remove(Name);
```

Comment:
Shouldn't we only remove TimeSeriesRangeResult's that are between 'from' and 'to' ? cc @garayx

Reply:
done